Library/Tracking/KSRAnalytics.swift

```diff
@@ -1503,10 +1503,7 @@ public final class KSRAnalytics {
     props["country"] = self.config?.countryCode
     props["display_language"] = AppEnvironment.current.language.rawValue
     props["device_type"] = self.device.deviceType
-    props["device_manufacturer"] = "Apple"
-    props["device_model"] = KSRAnalytics.deviceModel
```

Comment:
Doesn't look like we are using deviceModel for anything else in the file. I would delete this and double check everything is compiling fine.